### PR TITLE
fixes bug 1359927 - Move repo from github.com/mozilla to github.com/m…

### PR DESCRIPTION
**Pull request by peterbe**
…ozilla-services

### CODE REVIEW

File: tecken/templates/account/login.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2 class="panel-title">Links &amp; resources</h2>
       <div class="panel-body">
         <ul>
           <li>The <a href="https://telemetry.mozilla.org/">Telemetry Dashboards</a> cover the most common use cases for Telemetry data.</li>
-          <li>The <a href="https://github.com/mozilla/telemetry-analysis-service">source code</a> for this service.</li>
+          <li>The <a href="https://github.com/mozilla-services/telemetry-analysis-service">source code</a> for this service.</li>
```
Comment (willkg) [inline tecken/templates/account/login.html]: I think this replacement is a mistake.

File: tecken/templates/account/login.html [hunk 2]
```diff
@@ -29,7 +29,7 @@ <h2 class="panel-title">What is this?</h2>
         <p>The Telemetry Analysis Service gives direct access to Mozilla Telemetry data:</p>
         <ul>
           <li>Perform interactive analyses with <a href="http://spark.apache.org/">Apache Spark</a> and <a href="http://jupyter.org/">Jupyter</a>.</li>
-          <li>Run custom map-reduce analyses with the <a href="https://github.com/mozilla/telemetry-server/blob/master/docs/MapReduce.md">Telemetry MapReduce framework</a>.</li>
+          <li>Run custom map-reduce analyses with the <a href="https://github.com/mozilla-services/telemetry-server/blob/master/docs/MapReduce.md">Telemetry MapReduce framework</a>.</li>
```
Comment (willkg) [inline tecken/templates/account/login.html]: I think this replacement is a mistake.

Comment (peterbe) [inline tecken/templates/account/login.html]: Wow! Good catch! Rogue search-and-replace.

**Comment by willkg** [comment]
This looks good to me now.

**Pull request merged**